css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
```

Comment:
Oh, wait! This isn't on an `img` tag. So I'm guessing you were having issues with the image repeating itself? Maybe `background-repeat` would help?